packages/angular/.eslintrc.js

```diff
@@ -39,7 +40,12 @@ module.exports = {
     },
     {
       extends: ['@aws-amplify/amplify-ui/jest'],
-      files: ['**/__mocks__/**', '**/__tests__/**'],
+      files: [
+        '**/__mocks__/**',
+        '**/__tests__/**',
+        '**/*.spec.ts',
+        '**/*.jest.ts',
```

Comment:
Let's reconsider the approach to avoid adding these two lines. `'**/*.spec.ts'` is covered by the existing glob patterns (and if not we should figure out why not), and `'**/*.jest.ts'` only would capture _setup.jest.ts_ which is a config file that should be linted the same as other config files

Reply:
https://github.com/aws-amplify/amplify-ui/blob/main/packages/angular/projects/ui-angular/src/lib/components/authenticator/components/setup-totp/setup-totp.component.spec.ts isn't covered by the existing glob pattern